xbmc/interfaces/legacy/InfoTagMusic.h

```diff
@@ -406,6 +406,96 @@ namespace XBMCAddon
 #else
       String getLyrics();
 #endif
+
+#ifdef DOXYGEN_SHOULD_USE_THIS
+      ///
+      /// \ingroup python_InfoTagMusic
+      /// @brief \python_func{ getMusicBrainzTrackID() }
+      ///-----------------------------------------------------------------------
+      /// Returns the MusicBrainz Recording ID from music info tag (if present).
+      ///
+      /// @return [string] MusicBrainz Recording ID
+      ///
+      ///
+      ///-----------------------------------------------------------------------
+      ///
```

Comment:
please add a note to each new function that it's been added in v19
similar to:
https://github.com/xbmc/xbmc/blob/master/xbmc/interfaces/legacy/InfoTagMusic.h#L66

Reply:
Oops. Copy/paste gone wrong. Fixed now. Thanks!